charts/actions-runner/values.yaml

```diff
@@ -0,0 +1,59 @@
+image:
+  repository: summerwind/actions-runner
+  tag: 2.290.1
```

Comment:
Just a quick FYI that this doesn't appear to be a valid tag based on [the list of available ones](https://hub.docker.com/r/summerwind/actions-runner/tags?page=1&name=2.290.1).

Reply:
yeah, thanks. It was a mistake initially I put it as placeholder then didn't change.